package.json

```diff
@@ -14,19 +15,23 @@
     "semantic-ui-react": "0.71.5"
   },
   "devDependencies": {
+    "enzyme": "^2.9.1",
```

Comment:
@Gwillison415 if you wouldn't mind removing the `^` here and elsewhere in the dependencies, thanks!

Reply:
right sorry about that